create_publication.py

```diff
@@ -12,65 +12,55 @@
 import logging
 import timeit 
 from pathlib import Path
-from src.utils.data_connections import read_sql_file, get_df_from_server, make_database_connection
+from pyspark.sql import functions as F
+import requests
```

Comment:
Unused imports? Should they be used or can they be dropped?
```suggestion
from pyspark.sql import functions as F
```

Reply:
Accepted the suggestion and updated notebook